setup/scaling/caching/configure-redis-usage.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: single
+title:  "Configure Redis Usage"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker relies on its Redis cache for a number of reasons: caching
+infrastructure, storing live executions, returning pipeline definitions faster,
+etc... Each of these can be configured to your needs, whether it is to make
+Spinnaker more responsive, or reduce the load on a downstream dependency.
```

Comment:
The list of things we use Redis cache for does not map to the sections below.  That might be ok, but because you say "Each of these can be configured" it sounds like you're not delivering on a promise. 

It might be good to keep this list, all (or a representative list of) the things that we rely on Redis for, then instead of saying "Each of these...," say "You can configure Redis to optimizes some of these for scale..."